spybot.sls

```diff
@@ -6,9 +6,9 @@
 {% endif %}
 # Source: http://www.safer-networking.org/
 spybot:
-  '2.4':
```

Comment:
If the old version is no longer available, let's at least leave the `uninstall` data for it

Reply:
ok, not sure why I did not do this on this one. Have done this now.